.vscode/settings.json

```diff
@@ -14,7 +14,7 @@
     "yarn.lock": true
   },
   "editor.codeActionsOnSave": {
-    "source.fixAll.eslint": true
+    "source.fixAll.eslint": "always"
```

Comment:
vs code is annoyingly adding this each time, so lets commit it and see if there is any negative impact